ui/tests/helpers/clients.js

```diff
@@ -132,3 +135,769 @@ export async function dateDropdownSelect(month, year) {
   await click(dateDropdown.selectYear(year));
   await click(dateDropdown.submit);
 }
+
+export const ACTIVITY_RESPONSE = {
```

Comment:
this const in particular looks like it's a stubbed API response, is that right? if so, i'm wondering if we can solidify where we want to put stubbed API responses to make it clearer what API call it represents and where it's used. i can sort of infer here that this stub is supposed to represent part of the clients model, is that accurate? (semi-rhetorical question)

to make the connection/intended purpose of these kinds of consts more obvious we could create a new `/tests/helpers/stubs` directory for API stubs or group them within their respective test helpers (e.g., `/tests/helpers/clients-helpers`) with clear names, such as `ACTIVITY_RESPONSE_STUB`. regardless of the approach establishing a standard pattern will help reduce confusion and make it easier to identify their uses in the future 😀 

curious what your thoughts are!